uobtheatre/utils/test_utils.py

```diff
@@ -0,0 +1,24 @@
+def ticketDictListDictGen(ticketDictList):
+    ticketDict = {}
+    for ticket in ticketDictList:
+        ticketKey = (
+            ticket.get("seat_group_id"),
+            ticket.get("concession_type_id"),
+            ticket.get("seat_id"),
+        )
+        if ticketKey in ticketDict:
+            ticketDict[ticketKey] += 1
+        else:
+            ticketDict[ticketKey] = 1
+    return ticketDict
```

Comment:
Could we do something like this?

```suggestion
    ticketObjectList = map(Ticket, ticketDictList)
    return ticketListDictGen(ticketObjectList)
```

Reply:
You might actually need to use list comp here eg:
`ticketObjectList = [Ticket(**ticketDict) for ticketDict in ticketDictList]`

Reply:
Lovely suggestion there James!